backend/main_test.go

```diff
@@ -551,19 +553,196 @@ func TestFirebaseAuth(t *testing.T) {
 	req.Header.Set("Authorization", "Bearer reject")
 	resp = makeHTTPRequest(req, handler, t)
 	checkResponseStatus(http.StatusUnauthorized, resp, t)
+}
 
-	// Testing with unauthorised user domain
-	testAuth.AuthDomain = ""
+type UserAAuthClientStub struct {
```

Comment:
You probably don't need two different types here - you can just have a single `struct` which holds the user you want to return.

Reply:
Yep, should have caught that.